templates/activity/parts/activity-effects.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset>
+    <legend>{{ localize "DND5E.ACTIVITY.FIELDS.effects.label" }}</legend>
+    <multi-select name="appliedEffects">
```

Comment:
I just want to make sure I understand the UI here: If you choose an effect from this multi-select, the app re-renders and that effect appears immediately in the list below? Or it has to wait for some other re-render action before it appears?

Reply:
Yep, the change event on the `<mutli-select>` triggers form submission causing the newly selected effect to appear below.

Reply:
OK great, that sounds good to me then